### ISSUE TITLE
Invalid render using napeolon with Google-style docstrings

### ISSUE THREAD

**Issue by cblegare** [sphinx-doc/sphinx#4021]
Subject: Invalid render of return attribute type using napoleon

### Problem
- Warning thrown and invalid rendered HTML when using napeolon with Google-style docstrings

#### Procedure to reproduce the problem

```
git clone https://github.com/cblegare/sphinxbug
python -m venv .venv
.venv/bin/pip install -r requirements.txt
.venv/bin/python setup.py docs
```

#### Error logs / results

Logs

```
/home/bouchar1/workspace/tmp/test/thisbug/group.py:docstring of thisbug.group.Group.parent:0: WARNING: Inline strong start-string without end-string.
/home/bouchar1/workspace/tmp/test/docs/thisbug.rst:19: WARNING: Inline strong start-string without end-string.
```

Here is the generated HTML

```html
<dl class="class">
<dt id="thisbug.group.Group">
<em class="property">class </em><code class="descclassname">thisbug.group.</code><code class="descname">Group</code><a class="reference internal" href="_modules/thisbug/group.html#Group"><span class="viewcode-link">[source]</span></a><a class="headerlink" href="#thisbug.group.Group" title="Permalink to this definition">¶</a></dt>
<dd><p>Bases: <a class="reference external" href="https://docs.python.org/3.4/library/functions.html#object" title="(in Python v3.4)"><code class="xref py py-class docutils literal"><span class="pre">object</span></code></a></p>
<dl class="attribute">
<dt id="thisbug.group.Group.name">
<code class="descname">name</code><a class="headerlink" href="#thisbug.group.Group.name" title="Permalink to this definition">¶</a></dt>
<dd><p><a href="#id1"><span class="problematic" id="id2">**</span></a>Returns* – str* – The group name.</p>
</dd></dl>

<dl class="attribute">
<dt id="thisbug.group.Group.parent">
<code class="descname">parent</code><a class="headerlink" href="#thisbug.group.Group.parent" title="Permalink to this definition">¶</a></dt>
<dd><p><a href="#id3"><span class="problematic" id="id4">**</span></a>Returns* – str* – The group parent.</p>
</dd></dl>

</dd></dl>
```

#### Expected results

- strong tags (`**`) should be properly rendered

### Reproducible project / your project
- https://github.com/cblegare/sphinxbug

### Environment info
- OS: Archlinux
- Python version: Python 3.5.3
- Sphinx version: 1.6.3
- Other versions: (from `pip freeze`)
```
alabaster==0.7.10
Babel==2.5.0
certifi==2017.7.27.1
chardet==3.0.4
docutils==0.14
idna==2.6
imagesize==0.7.1
Jinja2==2.9.6
MarkupSafe==1.0
Pygments==2.2.0
pytz==2017.2
requests==2.18.4
six==1.10.0
snowballstemmer==1.2.1
Sphinx==1.6.3
sphinx-rtd-theme==0.2.4
sphinxcontrib-websupport==1.0.1
urllib3==1.22
```

#### EDIT

- fixed HTML sample

**Comment by cblegare** [comment]
Seams related to #4016

**Comment by tk0miya** [comment]
@RobRuana could you check this please?

**Comment by ksunden** [comment]
Note that, while I think the issue where these docstrings do not render as expected here is causing at least part of #4016 (which I filed), the docstrings in the example here are not exactly following google style: They do not have summary lines. Adding a summary line makes them render as expected.

Summary lines are required by google [style guide](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments).

```
A doc string should be organized as a summary line (one physical line) terminated 
by a period, question mark, or exclamation point, followed by a blank line, 
followed by the rest of the doc string starting at the same cursor position as the 
first quote of the first line.
```

**Comment by goodmami** [comment]
I commented in #4016, but it seems that the issue is more general and is relevant here. If I use a colon in inline formatting, I get the error, regardless if the formatting is an inline literal, strong, or emphasis.

**edit**: sorry, I realized that this is in fact different from my problem (and #4016). I read the problem description as meaning that the `**` strong formatting was explicitly used in the docstring, but that is not the case. @ksunden's comment above seems to describe the real issue here well.
